static/llms.txt

```diff
@@ -1494,8 +1494,6 @@ let errored = $state(false);` to make this demo:
 {/snippet}
 ```
 
-## Addendum: more components
-
```

Comment:
i'm not sure if you realized yet but you're supposed to generate this through `node scripts/generate-demos.ts` or `deno --allow-all scripts/generate-demos.ts`, and if you want to modify the structure of things you should modify that script and rerun it